content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,210 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+## Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves <abbr>params</abbr> keys and values from a <abbr>DVC project</abbr>
+and returns a JSON structure such as:
```

Comment:
Isn't it a Python dict and not a JSON structure?

Reply:
My bad. Fixed.